server/src/main/java/com/google/moviestvsentiments/account/Account.java

```diff
@@ -0,0 +1,78 @@
+package com.google.moviestvsentiments.account;
+
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import java.util.Objects;
+
+/**
+ * A record in the accounts database table.
+ */
+@Entity
+public class Account {
+    
+    @Id
+    private String accountName;
+    private long timestamp;
```

Comment:
Prefer using actual time types instead of primitive long. 

https://drive.google.com/file/d/1-xgevTFxo_5IsR6GUMAQy71ZERjykSBx/view?usp=sharing
https://drive.google.com/file/d/1s1sYKYWq9l7ORtt1ZRuu4GplUABQ-M-5/view?usp=sharing